assume/units/powerplant.py

```diff
@@ -175,10 +178,13 @@ def calculate_energy_operational_window(
 
         min_power, max_power = self.calculate_min_max_power(start, end)
 
+        if self.id == "pp_02":
+            logger.debug(f"min_power: {min_power}")
+
         # adjust for ramp down speed
-        min_power = max(-self.ramp_down, min_power)
+        min_power = max(current_power - self.ramp_down, min_power)
         # adjust for ramp up speed
-        max_power = min(self.ramp_up, max_power)
+        max_power = min(current_power + self.ramp_up, max_power)
```

Comment:
the change from relative changes to absolute changes breaks compatibility for multiple markets, as this would bid the already bidded power on multiple markets for the same interval.

Reply:
Here you are mixing up bidding and state calculations. An operational window should reflect the possible states of my power plant in absolute terms. The bidding strategy in the following step takes care of the relative aspect. Such relative formulation at the operational window calculation completely broke the bidding strategies.

Reply:
Which bidding strategy takes care of this? Naive does not, flexable does not.

With your change, you would bid max_power on the EOM market and bid the same max_power additionally on the CRM_pos? Or am I missing something?

Reply:
flexable does, for EOM. it should be also fixed for CRM.

Reply:
See #100 to see how i would expect working (current behavior) - your change would create a min_power of 400 whysoever :shrug:

Reply:
ok, jetzt sehe ich was du meinst, hab es korrigiert, dein test ist auch in diesem commit drin. Ich glaube, jetzt funktioniert es